components/src/main/java/uk/gov/hmrc/components/organism/editable/EditableListViewAdapter.kt

```diff
@@ -51,10 +53,16 @@ class EditableListViewAdapter(
         RecyclerView.ViewHolder(binding.root) {
 
         fun bind(result: EditableListView.EditableItem, position: Int) = with(binding) {
+            val spacing16 = root.resources.getDimensionPixelSize(R.dimen.hmrc_spacing_16)
+            val spacing4 = root.resources.getDimensionPixelSize(R.dimen.hmrc_spacing_4)
+            val spacing0 = root.resources.getDimensionPixelSize(R.dimen.hmrc_spacing_0)
+
             columnOne.text = result.name
             columnTwo.text = result.value
             iconButton.text = result.buttonText
+            iconButton.setPadding(spacing16, spacing4, spacing0, spacing4)
```

Comment:
If we are changing the padding value of the button it would be better to do it in the xml.
However, can we instead adjust remove the inner padding of the container, rather than the button itself. It makes the touch state look off. The button should keep it's padding and we should remove it from the right of the card, then the button touch state will extend to the edge of the card, and won't make the text look off centre. To achieve this I think you'll need to remove the horizontal padding from the recycler view, and add it to each item in the rows.
<img width="354" alt="Screenshot 2023-01-27 at 13 43 51" src="https://user-images.githubusercontent.com/7998113/215101299-d48f49c3-2b31-4dee-9cfc-8121cf0ecf5a.png">